consensus-types/wrapper/beacon_block.go

```diff
@@ -117,29 +117,45 @@ func WrappedBeaconBlockBody(i interface{}) (interfaces.BeaconBlockBody, error) {
 func BuildSignedBeaconBlock(blk interfaces.BeaconBlock, signature []byte) (interfaces.SignedBeaconBlock, error) {
```

Comment:
I guess I am confused, why do we still need the wrapper methods ? Arent we shifting away from them

Reply:
I want to remove them in the next PR